src/Libraries/CoreNodes/CoreNodes.csproj

```diff
@@ -40,6 +40,11 @@
       <Project>{c0d6dee5-5532-4345-9c66-4c00d7fdb8be}</Project>
       <Name>DesignScriptBuiltin</Name>
     </ProjectReference>
+    <ProjectReference Include="..\DynamoUnits\UnitsCore.csproj">
+      <Project>{6e0a079e-85f1-45a1-ad5b-9855e4344809}</Project>
+      <Name>Units</Name>
+      <Private>False</Private>
```

Comment:
also, does this duplicate the units dll?

Reply:
Do you mean copy local?  Not sure what you are asking.

Reply:
yeah I want to make sure we are not adding extra binaries to the nodes folder etc.

Reply:
Should be good
![image](https://github.com/DynamoDS/Dynamo/assets/2145751/e63ae169-134a-4f99-98c1-27a7542f0b93)